app/util/data_preparation/bitbucket_prepare_data.py

```diff
@@ -79,11 +79,20 @@ def __get_prs(bitbucket_api):
             for pr in prs['values']:
                 # filter PRs created by selenium and not merged
                 if 'Selenium' not in pr['title']:
-                    if pr['properties']['mergeResult']['outcome'] != 'CONFLICTED':
-                        repos_prs.append([repo['slug'], repo['project']['key'], pr['id'],
-                                          pr['fromRef']['displayId'], pr['toRef']['displayId']])
+                    # Some PRs do not have 'mergeResult' in properties, force get merge result status otherwise.
+                    if 'mergeResult' in pr['properties']:
```

Comment:
`                    if 'mergeResult' in pr['properties'] and pr['properties']['mergeResult']['outcome'] == 'CONFLICTED':
                        pr['with_conflict'] = True
                    elif bitbucket_api.check_pull_request_has_conflicts(project_key=repo['project']['key'], 
                                                                          repo_key=repo['slug'], pr_id=pr['id']):
                        pr['with_conflict'] = True
                        
                    if pr['with_conflict']:
                        print(f"Pull request {pr['links']['self'][0]['href']} has a conflict.")
                    else:
                        repos_prs.append([repo['slug'], repo['project']['key'], pr['id'],
                                          pr['fromRef']['displayId'], pr['toRef']['displayId']])`